inc/class-s3-uploads-stream-wrapper.php

```diff
@@ -17,7 +17,34 @@ public static function register( Aws\S3\S3Client $client)
 		static::$client = $client;
 	}
 
+	// Override
+	public function stream_flush() {
+
+		// Add some HTTP headers which can be defined in wp-config.php
+
+		// Expires:
+		if ( defined( 'S3_UPLOADS_EXPIRES' ) ) {
+			$this->params[ 'Expires' ] = S3_UPLOADS_EXPIRES;
```

Comment:
This will create an array implicitly and throw a strict standards warning (IIRC). Can we add `protected $params = array()` as a property to fix this please?